governance.md

```diff
@@ -159,10 +159,10 @@ Working Groups operate transparently and must follow the Code of Conduct.
 ## 12. Amendments
 
 Amendments to this document require:
-
-- A proposal posted publicly.
-- A 14-day comment period.
-- Approval by two-thirds of the Steering Committee.
+- A proposal posted publicly as an [Issue in the governance repository](https://github.com/OpenIntegrationEngine/governance/issues)
```

Comment:
Can we just change this to a Pull Request instead of an issue? I don't have a problem with including the `amendment` label.

Unless someone can tell me why it's better to propose a change in an issue and then copy the proposal to a PR.

Reply:
I'm not against someone opening an issue first if it seems warranted. I'm just against it as a requirement, because the PR is the thing that needs to be reviewed and get approval, not the issue.